docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
             - "3000:3000"
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
-            - ".rustwide:/home/cratesfyi/rustwide"
+            - ".rustwide-docker:/home/cratesfyi/rustwide"
```

Comment:
Unrelated: can you change the path on the container to be `/opt/docsrs/rustwide`? You'll also need to change the environment variable below.